cmd/thanos/compact.go

```diff
@@ -221,11 +221,6 @@ func runCompact(
 		indexCacheDir   = path.Join(dataDir, "index_cache")
 	)
 
-	if err := os.RemoveAll(downsamplingDir); err != nil {
```

Comment:
this is not removed , it is just moved inside the .Compact() method. This makes it consitent with all the how the other components handle removing the tmp files.

Reply:
actually it is not moved as it is redundant because the compactor removes these before every compaction in 

https://github.com/thanos-io/thanos/blob/3646e13fd8763a0fe4de53863cbd037d29022ddd/pkg/compact/compact.go#L1044-L1047

Reply:
Yes, but  `c.compactDir`  is different then `downsamplingDir` :thinking:

Reply:
correction It is a duplicate of 
https://github.com/thanos-io/thanos/blob/95757ea641511d84a1e9646031d69a4edf7845ef/cmd/thanos/downsample.go#L134-L136

Reply:
Yes but that would be cleaned only after (!) compaction, so long time. I am 99% sure we need to keep this.